internal/dotnet/identify.go

```diff
@@ -0,0 +1,38 @@
+package dotnet
+
+import (
+	"github.com/spf13/afero"
+
+	"github.com/zeabur/zbpack/internal/utils"
+	"github.com/zeabur/zbpack/pkg/plan"
+	"github.com/zeabur/zbpack/pkg/types"
+)
+
+type identify struct{}
+
+// NewIdentifier returns a new Dotnet identifier.
+func NewIdentifier() plan.Identifier {
+	return &identify{}
+}
+
+func (i *identify) PlanType() types.PlanType {
+	return types.PlanTypeDotnet
+}
+
+func (i *identify) Match(fs afero.Fs) bool {
+	return utils.HasFile(fs, "Program.cs", "Startup.cs")
```

Comment:
Can it support other project types than `ASP.NET`?

Reply:
I've tested it for ASP.NET Web Apps(Blazor & MVC), ASP.NET Web APIs, and ASP.NET Microservices. I haven't tested with .NET Framework apps and it won't work atm.  I have sample projects that I created for them, do you want me to add it to the `tests` folder?

Keep in mind that I did not take into consideration more complex applications with multiple modules and when the `.csproj` isn't present in the root. We need a way to parse the `.sln` file, read the modules in the project, and add it to the docker template. I'm open to any ideas if you know of a good way to do this.